datasets/camvid.py

```diff
@@ -90,21 +90,21 @@ def __init__(self, root_dir, d_type, classes=None, download=True, transform=None
                         break
                     img_crop = copy.deepcopy(img[y_start:y_end, x_start:x_end, :])
                     lbl_crop = copy.deepcopy(lbl[y_start:y_end, x_start:x_end])
-                    if fold_ratio == 1:
-                        self.img_list.append(img_crop)
-                        self.lbl_list.append(lbl_crop)
-                    else:
-                        img_crop_folded = None
-                        for i in range(fold_ratio):
-                            for j in range(fold_ratio):
-                                img_crop_subsample = img_crop[i::fold_ratio, j::fold_ratio, :]
-                                if img_crop_folded is not None:
-                                    img_crop_folded = np.concatenate((img_crop_folded,
-                                                                      img_crop_subsample), axis=2)
-                                else:
-                                    img_crop_folded = img_crop_subsample
-                        self.img_list.append(img_crop_folded)
-                        self.lbl_list.append(lbl_crop)
+                    # if fold_ratio == 1:
+                    self.img_list.append(img_crop)
```

Comment:
May it better be to remove these parts instead of commenting them out

Reply:
It has already done in one of the latest commits. Did you write this comment on a previous commit?

Reply:
Yes many thanks, the last commit addresses all.